addons/website/static/src/scss/website.scss

```diff
@@ -774,3 +786,36 @@ table.table_desc tr td {
         background-repeat: no-repeat;
     }
 }
+
+// Scroll down button
+.o_scroll_button {
+    animation: o-anim-heartbeat 2.6s ease-in-out 1s infinite;
+    width: 50px;
+    height: 50px;
+    display: none;
+
+    &:focus {
+        outline: none;
+    }
+
+    &:hover {
+        text-decoration: none;
+        animation-iteration-count: 1;
+    }
+}
+
+// Hide button when not using a min-height
+.o_full_screen_height, .o_half_screen_height {
+    .o_scroll_button {
+        display: flex;
+    }
+}
+
+// Attention keeper for the "scroll down" top-banner button
+@keyframes o-anim-heartbeat {
+    0%, 14%, 35% { transform: scale(1); }
+    7%, 21% {
+        transform: scale(1.3);
+        background-color: rgba(theme-color('primary'), 0.8);
+    }
+}
```

Comment:
Your shiny editor does not even add a new line at the end of files :'(

Reply:
I had to turn that off because I got chewed for removing whitespace at end of lines while making unrelated changes, since that makes forward porting harder, and addind newlines at end of files is done by the same module so I just turned it off.

Reply:
Please be careful to not add spaces at the end of lines then, it is so easy to do that by mistake. I personally just rarely do `git add .` but always `git app -p .` so that I can review what I stage and be sure to not stage lines I did not edit by myself.